content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,1141 @@
+|Tags|`spectro-cloud-education, app:hello-universe, terraform_managed:false`|
+
+Click **Next** to continue.
+<br /> 
+
+### Parameters
+
+The **Parameters** section offers you another opportunity to change the profile configurations. For example, clicking on the **BYOS Edge OS 1.0.0** layer allows you to configure the `system.registry`, `system.repo`, and other available attributes. 
+
+Use the default values for all attributes across all layers and click **Next**.
+<br />
+
+### Cluster config
+
+Provide the Virtual IP (VIP) address for the host cluster to use during the cluster configuration process. You can optionally select an SSH key to remote into the host cluster and provide a Network Time Protocol (NTP) server list, if available, with you.  
+
+Click **Next** to continue.    
+<br />
+
+### Nodes config
+
+In this section, provide details for the master pool. 
+
+|**Field** | **Value for the master-pool**| 
+|---| --- | 
+|Node pool name| master-pool | 
+|Allow worker capability| Checked | 
+|Additional Labels (Optional) | None |
+|[Taints](/clusters/cluster-management/taints/)|Off|
+|Pool Configuration > Edge Hosts | Choose one of the registered Edge hosts.<br />Palette will automatically display the Nic Name for the selected host. |
+
+The screenshot below shows adding an Edge host to the master pool.
+
+![Screenshot of adding an Edge host to the master pool.](/tutorials/edge/clusters_edge_deploy-cluster_add-master-node.png)
+
+
+Similarly, provide details for the worker pool, and add the remaining two Edge hosts to the worker pool. Suppose you do not have Edge hosts remaining; in that case, you can remove the worker pool. Your master pool has the worker capability already. 
+
+|**Field** | **Value for the worker-pool**| 
+|---| --- | 
+|Node pool name| worker-pool | 
+|Additional Labels (Optional) | None |
+|Taints|Off|
+|Pool Configuration > Edge Hosts | Choose one or more registered Edge hosts. |
+
+The screenshot below shows adding two Edge hosts to the worker pool.
+
+![Screenshot of adding Edge hosts to the worker pool.](/tutorials/edge/clusters_edge_deploy-cluster_add-worker-node.png)
+
+Click **Next** to continue.    
+<br /> 
+
+### Settings 
+
+This section displays options for OS patching, scheduled scans, scheduled backups, cluster role binding, and location. Use the default values, and click on the **Validate** button.      
+<br /> 
+
+### Review
+
+Review all configurations in this section. The **Review** page displays the cluster name, tags, node pools, and layers. If everything seems good, click on the **Finish Configuration** button to finish deploying the cluster. Deployment may take up to *20 minutes* to finish. 
+
+While deployment is in progress, Palette displays the cluster status as **Provisioning**. While you wait for the cluster to finish deploying, you can explore the various tabs on the cluster details page, such as **Overview**, **Workloads**, and **Events**. 
```

Comment:
Let's add a youTube video here. Justin might be close with his video, otherwise, find an Edge video from our official YouTube channel.